PSReadLine/PublicAPI.cs

```diff
@@ -29,6 +29,8 @@ public interface IPSConsoleReadLineMockableMethods
             Task<List<PredictionResult>> PredictInput(Ast ast, Token[] tokens);
             void OnCommandLineAccepted(IReadOnlyList<string> history);
             void OnSuggestionAccepted(Guid predictorId, string suggestionText);
+            object GetDynamicHelpContent(string commandName, string parameterName, bool isFullHelp);
+            void WriteToPager(string content, string regexPatternToScrollTo);
```

Comment:
How about rename this method to `RenderFullHelp`? Or some another similar name.

Reply:
Renamed. Like `RenderFullHelp`